insertion_sort/C/insertionsort.c

```diff
@@ -0,0 +1,30 @@
+// Program implementing insertion sort using array in C
+#include<stdio.h>
+// Function to sort an array using insertion sort
+void insertionsort(int ar_size,int *ar) {
+    int temp, j;
+    for(int i = 1; i < ar_size; i++) {
+        j = i;
+        temp = ar[j];
+    /* Shift elements by one position ahead of its current position
+       that are greater than temp(the element being sorted).*/
+        while(j > 0 && temp < ar[j-1]) {
```

Comment:
Missing space before ( in while(  [whitespace/parens] [5]

*Origin: CPPLintBear, Section: `all.cpp`.*